test/test_public_bindings.py

```diff
@@ -308,6 +308,7 @@ def onerror(modname):
             "torch.onnx._internal.exporter._reporting",
             "torch.onnx._internal.exporter._schemas",
             "torch.onnx._internal.exporter._tensors",
+            "torch.onnx._internal.exporter._torchlib.ops",
```

Comment:
@wconstab this is failing because this relies on the "pulp" package to run which is not a dependency. Why do we depend on this to even import the files / any details where I can read about why we are making this a dependency?

Reply:
It's for the file below in `torch.distributed._tools.sac_ilp` which was built as an optional tool for memory estimation for FSDP and uses `pulp` as an ILP solver. But without `pulp` the tool doesn't work

https://github.com/pytorch/pytorch/blob/3917053f63b75f14e3cb2f53805fad4ade5363df/torch/distributed/_tools/sac_ilp.py#L11